console/console-init/ui/src/modules/iot-project-detail/Routes.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { SwitchWith404, LazyRoute } from "use-patternfly";
+import { Redirect } from "react-router";
+
+const getCertificates = () => import("modules/address/AddressPage");
+
+export const Routes = () => (
+  <SwitchWith404>
+    <Redirect path="/" to="/address-spaces" exact={true} />
+    <LazyRoute
+      path="/iot-projects/{namespace}/{projectname}/certificates"
```

Comment:
I think this route path is wrong, you are calling namespace and projectname as local variable but it should be as below .
 path="/iot-projects/:namespace/:projectname/certificates"